spec/flow-api.yaml

```
@@ -563,18 +578,72 @@ components:
           $ref: "#/components/schemas/InfoTypeEnum"
         reason:
           type: string
-        code:
-          type: integer
-        seq_no:
-          type: integer
-        quality:
-          type: string
       required:
         - message
         - type
         - reason
 
+    StatusUpdateInfo:
```

Adding a ~`name`~ `title` field to enum variants helps with the rendering
```suggestion
    StatusUpdateInfo:
      title: StatusUpdateInfo
```

EDIT sorry it should be `title` not `name`